packages/core/src/sync-store/store.test.ts

```diff
@@ -2661,7 +2662,7 @@ test("getEvents filters on toCheckpoint (inclusive)", async (context) => {
 
   const ag = syncStore.getEvents({
     sources: [sources[0]],
-    fromCheckpoint: zeroCheckpoint,
+    fromCheckpoint: createBlockCheckpoint(rpcData.block2.block, false),
```

Comment:
For my understanding: Now, it's no longer OK/safe to pass zeroCheckpoint as fromCheckpoint?

Reply:
it is fine, just does a ton of extra queries